src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { wait } from '../../utils/fetchClient';
+import { Error } from '../../types/Error';
+
+type Props = {
+  errors: Error[];
+  setErrors: (cb: (prev: Error[]) => Error[]) => void;
+};
+
+export const Notification: React.FC<Props> = ({ errors, setErrors }) => {
+  const [isHidden, setIsHidden] = useState(false);
+  const { title, isImportant } = errors[0];
```

Comment:
This line may cause an error when the errors array is empty. You should check if there is an error at index 0 before destructuring.

Reply:
<img width="473" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/17066980/19542df9-1a73-4684-8ced-989efcec952c">